documentation/node-specs.md

```diff
@@ -0,0 +1,53 @@
+#Ethereum Node Specs
+It is possible to sync and run an Ethereum node in several different ways based on the sync strategy being used, and 
+the pruning configuration. This document aims to explain those differences, and what is required for use with 
+VulcanizeDB.
+
+VulcanizeDB currently depends on a (custom Geth implementation)[https://github
+.com/makerdao/go-ethereum/tree/allow-state-diff-subscription]. The key piece that this implementation adds is a way 
+to subscribe to state diffs (the change in state from block to block). These state diffs are used to build up the 
```

Comment:
This is pretty nitpicky but I wonder if we might want to start to distinguish between state vs storage diffs - I think at this point we're really only able to access the latter (e.g. modifications to state on an individual contract account) as opposed to broader changes like new addresses appearing on the network/eth balances/etc

Reply:
That's a good point - I had thought of "state diffs" as changes to the state trie for an individual contract, which I think we would be able to access via diffs, but we're not really using those specifically in vdb, also you're right that we're not able to get broader changes. I'll audit this doc and make sure to refer to these diffs as storage diffs. 👌